tools/apt-get-install-tizenrt.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+sudo apt-get update
+sudo apt-get install gcc-arm-none-eabi
+arm-none-eabi-gcc --version
```

Comment:
This invocation has two results:
 - verifies the 'arm-none-eabi-gcc' command is available and can be executed (so we have the compiler)
 - pushes compiler version into logs, for further reference